Mod2/Labs/Week5/ManyToManyWithAnORM.md

```diff
@@ -0,0 +1,8 @@
+## Lab Objectives
+* Practice building a many-to-many relationship with an ORM
+
+In today's lab, we're going to add to our library application and database. Before beginning make sure you have the library application open.
+
```

Comment:
I think this lab works fine as-is; but, as an additional push/stretch, we could ask students to think about the data that a library tracks, and see if they can identify a many-to-many.  Then after a brainstorm, have them all create the author/book relationship.

Reply:
I like this idea of having students think of many-to-many relationships they could implement in the library db. I'm thinking I'll have them start with the book/author because they already did that with pure SQL and then they will have an ORM version of creating all the tables from the first part of the mod. 

But I think that will be very quick, so thinking of having them also brainstorm other possible many-to-many relationships and implement one of those after.